generators/app/templates/src/app/screens.ejs

```diff
@@ -3,8 +3,14 @@
 //  in favour of setting the initialRoute as a prop of the navigator
 
 <%_ if(features.drawerandroid || features.drawerios) { _%>
+<%_ if(features.login) { _%>
+import React, { PureComponent } from 'react';
+<%_ } else { _%>
 import React from 'react';
 <%_ } _%>
+<%_ } else if(features.login) { _%>
+import { PureComponent } from 'react';
```

Comment:
🤔  At first glance these lines are not necessary, since this import is covered by the previous `if (features.login)` block